script/bin/operator.rs

```diff
@@ -45,13 +73,17 @@ sol! {
         function commitHeaderRange(bytes calldata proof, bytes calldata publicValues) external;
     }
 }
-struct VectorXOperator {
-    pk: SP1ProvingKey,
+
+type SP1VectorInstance<P, N> = SP1Vector::SP1VectorInstance<(), P, N>;
+
+struct SP1VectorOperator<P, N> {
+    pk: Arc<SP1ProvingKey>,
     vk: SP1VerifyingKey,
-    contract_address: Address,
-    rpc_url: Url,
-    chain_id: u64,
-    use_kms_relayer: bool,
+    signer_mode: SignerMode,
+    tree_size: u32,
+    fetcher: RpcDataFetcher,
+    prover: NetworkProver,
```

Comment:
```suggestion
    prover: Arc<NetworkProver>,
```

This also is several GB.

Reply:
Actually nothing needs to be in an Arc anymore because they don't cloned, I'll also remove it from PK